addon/components/ember-popper-base.js

```diff
@@ -89,7 +120,7 @@ export default class EmberPopperBase extends Component {
 
   @action
   update() {
```

Comment:
since there is already an `update()` method on Poppers, perhaps we should rename this to `scheduleUpdate()` incase someone has an explicit need for `update()`. Thoughts?